module_intros/density.CorrelationFunction-Orientations.ipynb

```diff
@@ -0,0 +1,265 @@
+{
```

Comment:
<p>Important API question. Should we be doing the conjugation automatically? I actually think that we should, because the definition of the correlation function should always give a real value. If you agree, we need to resolve this before the 2.0 release since that's a meaningful API change.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>